MAUI/AIAssistView/working-with-aiassistview.md

```diff
@@ -112,6 +112,90 @@ public class ViewModel : INotifyPropertyChanged
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/getting-started-with-.net-maui-aiassistview)
 
+### StopResponding Text
+
+The `SfAIAssistView` control allows you to customize the StopResponding text using the [StopRespondingText](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.AIAssistView.SfAIAssistView.html#Syncfusion_Maui_AIAssistView_SfAIAssistView_StopRespondingText) property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3" %}
+         <syncfusion:SfAIAssistView x:Name="sfAIAssistView"
+                                    EnableStopResponding="True"
+                                    StopRespondingText="Close Response"/>  
+
+{% endhighlight %} 
+
+{% highlight c# hl_lines="6" %} 
+
+    SfAIAssistView sfAIAssistView; 
+    public MainPage() 
+    { 
+        InitializeComponent(); 
+        this.sfAIAssistView = new SfAIAssistView();
+        this.sfAIAssistView.StopRespondingText = "Terminate Response";
+        this.sfAIAssistView.EnableStopResponding = true;
+        this.Content = sfAIAssistView; 
+     } 
+
+{% endhighlight %}
+{% endtabs %}
+
+![Stop Responding Text in .NET MAUI AI AssistView](Images/working-with-aiassistview/maui-aiassistview-stopresondingtext.png)
+
+### StopResponding Customization
+
+The `SfAIAssistView` control allows you to fully customize the Stop Responding appearance by using the [StopRespondingTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.AIAssistView.SfAIAssistView.html#Syncfusion_Maui_AIAssistView_SfAIAssistView_StopRespondingTemplate) property. This property lets you define a custom layout and style for the StopResponding.
```

Comment:
The `SfAIAssistView` control allows you to fully customize the `StopResponding` view appearance by using the [StopRespondingTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.AIAssistView.SfAIAssistView.html#Syncfusion_Maui_AIAssistView_SfAIAssistView_StopRespondingTemplate) property. This property lets you define a custom layout and style for the StopResponding UI.